src/actions/index.js

```diff
@@ -0,0 +1,3 @@
+export * as columnActions from './columnActions';
+export * as scrollActions from './scrollActions';
+export { initialize, propChange } from '../reducers';
```

Comment:
Created a new barrel file to re-export actions so that they can be imported from '../actions'.